ebean-core/src/main/java/io/ebeaninternal/server/persist/BatchControl.java

```diff
@@ -267,8 +270,8 @@ public void clear() {
   private void flushBuffer(boolean reset) throws BatchedSqlException {
     flushQueue(queues[0]);
```

Comment:
We were unable to find a use of the queue at position 0. Maybe this one is unused?

Reply:
Ah ok ... hmm, I need to look at that.